data/management/commands/update_iati_publishers_table.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from sqlalchemy import distinct, create_engine, MetaData, select, Table
+from sqlalchemy.sql import func
+
+DATA_SCHEMA = "repo"
+TXNS_TABLE_NAME = "iati_transactions"
+PUBLISHERS_TABLE_NAME = "iati_publishers"
+
+class Command(BaseCommand):
+    help = 'Downloads CSV files from git repo'
+
+    def handle(self, *args, **kwargs):
+        self.updateIatiPublishersTable()
+
+    def updateIatiPublishersTable(self):
+        engine = create_engine('postgresql://analyst_ui_user:analyst_ui_pass@db:5432/analyst_ui')
```

Comment:
Using SQLalchemy within a management command feels a bit weird. Would it be possible to use the utilities from our db_manager? E.g. https://github.com/devinit/ddw-analyst-ui/blob/develop/data/db_manager.py#L6 ?

Reply:
Actually all iati_transaction*.py scripts seem to take the route of SQLalchemy, so shall we have to update all of them?

Reply:
Moved it to the `data_updates` folder and updated the `iati_transactions.sh` script consequently.